plot_pixel_values.py

```diff
@@ -22,41 +24,103 @@ def get_opt():
     parser.add_argument('--fit-scaling',
                         action='store_true',
                         help='Fit the dark and scale factor (default=False)')
-
+    parser.add_argument('--plot-fits',
```

Comment:
Do we really need that plot?  It will be misleading in the case of flickering pixels (which will likely happen).  This tool should have the minimum clean output that let's us answer the question: can we raise the setpoint now?  To say yes we just need to know that our dark model is roughly correct at these higher temperatures.